app/services/hyrax/manifest_builder_service.rb

```diff
@@ -0,0 +1,93 @@
+require 'iiif_manifest'
+
+module Hyrax
+  # A class responsible for converting a Hyrax::Work like thing into a IIIF
+  # manifest.
+  #
+  # @see !{.as_json}
+  class ManifestBuilderService
+    # @api public
+    #
+    # @param presenter [Hyrax::WorkShowPresenter] the work presenter from which
```

Comment:
the documentation above says this works for `Hyrax::Work`-like objects. what's the correct type signature?

Reply:
i think the answer to this really is `Hyrax::WorkShowPresenter`, at least for now. maybe we need to edit the top level class documentation?